samples/nrf_desktop/configuration/app_nrf52840_pca20041.conf

```diff
@@ -42,22 +42,8 @@ CONFIG_PWM_NRF5_SW=n
 CONFIG_PWM_NRFX=y
 
 CONFIG_PWM_0=y
-CONFIG_PWM_0_NAME="PWM_0"
```

Comment:
Just a comment here: The names are still set through Kconfig - no change here. So if there was a reason for you to write this name in your .conf file, then that reason should still be there - but as far as I can tell, you were just using the default name of the driver so its OK to remove this.

Reply:
Actually this is set with DT_NORDIC_NRF_PWM_PWM_0_LABEL, so I can use it.

Reply:
The label comes from dtsi file

Reply:
Discussed offline, leaving Kconfig name which is the one actually used.